bristol-interns-35751983eaf5.json

```diff
@@ -0,0 +1,12 @@
+{
+  "type": "service_account",
+  "project_id": "bristol-interns",
+  "private_key_id": "35751983eaf54fe5a11aea92ee3a317ef77bce19",
+  "private_key": "-----BEGIN PRIVATE KEY-----\nMIIEvgIBADANBgkqhkiG9w0BAQEFAASCBKgwggSkAgEAAoIBAQC18WFx0UwY6HZa\nxmjzmzs1tuUXF228EHCVH3JuGc6VKf15BLtCaVebTtJNiHTsM+rTgTVTHJrocziM\n6TlZm5fEodHvS6Qio9zrGkwf5vF0PCyaQfQs9uxeJ2qzAOpytWj8+ohSjbHJz4lj\ne06UGkL5HzpmQYDPMBkAVDjt4uGkzZcs1P96KVuRk3BHTgPoQsVR2pz22tE8zCuw\nqlMJpngrdU8p58V040c26ha2w8zP+6hJ/HGbX0A8WvTGwW3bgsxX9X5bNRCH7N40\ntrQgc8khTyX9dlI6cC8gH1qQdwWdCeyN4E98nWpSNpnM7TxdPkm2Jk/GVndVGp2s\n/5y0vphbAgMBAAECggEACol5tsFD0nd4D0bJYj5dvf0jJGS3drpPAtvo4SZ+d9YQ\nN3eGLtDyAW0dsd7SwrE5CtnDUFZqREcUAdkp+J4RupISmfS8A0ORMk56FSjnV+96\nN87VCNueEc/flpFZgp4t/+S0i29oWgM6djf4c4xXz5y24aAuiRFEiWfXSjOnFMe4\nLeNo0H6XPLnkxycNX3SZ6NuoVik0meej+iigIBVuB2bDePpwICcpKRvIUyIUnuzf\nGim0UhSLrOJFrUnZe8Zulnc8jM0brT9je5BtJ40lc5hnIar9autIDkhSDzLE8JwG\nzvhc9nkHJfCCnKAIJiZNFk3Gp58zTXtZpjSzbC1wQQKBgQDx6azoAaUHk1mBToEz\nk/l7XgiOcJgusAyVL60AZz0uWDAdbat/HkBGE4d6OTPh8EXyWSpHjL12jOVfPEDE\nKr8r6109yzoiHTXa6+EvyZdivhE8cK+wk+/ZJULHcrTsI2QoAM2xLNObwOUoOp9Q\n6mF+womJ2JhClljZfxSiqyu8wQKBgQDAibOYRQ9ssQXc9ca94bGBbm/fY+0gZp+x\nXKDDuyk9FsQsl8kiLhS8HtYWV20yvVhtiFtJq9RKE/X3xn799W6mq4ktKyJj2qQT\n+yxoycXFVqwsyoRBrXgB+ADfXYImMoaLdNBA+Wvggtb84HeeNUkZSSVkeQiSp5P+\naszRk52wGwKBgQCnj2XWZZ07fBKZABl58lrx2pYuo0qlvAdbJmS8TMYR3b+mApqx\nN/DpGQnT3N9U9ipczOn+GTv6a2lM590sgzdbYnF5JdUkX/s3of6eLRIQsPobVSo8\nMpwFWkvA/nDWx0TOeJAA0r3Xump/zOshDtdtB65grof4R09Phzhx3AI1gQKBgAp9\ntlZ+tNd0UeB3V+vyx0xJKzld5YbVcLAmXvArW5Tjj67qWmBNaH3TpFC1VNLl9FKL\nkoXSIhAjREzK+RJ3dB8Ky7+1UPtl53jbP9E55VfaYClUo548iIR68fk5zCSurhwd\nMlfH/3PNTU+cH4h0WNZS4QG3gpsTHIJJaX2l8HqvAoGBAMcXeU6ec9k1qfhyyxdq\nwNBBcdb6SgU0z3Tqq2ZEG/uB6bXptxyIJ3X5q4iKmmsQH0IN1DuBS6hN/UZTuTKh\nZKh12vI9k1aFIcVJpW5qmnTb8LHnrIUMW59UU5oUylVcqipEUs3GXSYdATzQD+G9\nR4tn/yBbIATmdhiLumA7fCex\n-----END PRIVATE KEY-----\n",
```

Comment:
Are you commiting secret/private information?

Reply:
yeah, this is the kind of thing that scrapers search public GitHub repositories for.

you should assume that $INSERT_COUNTRY is using your credentials to buy compute on Google Cloud now.

Reply:
????? Should I be concerned...

Reply:
deleted the file from branch, then deleted the key and made a new key which is in gitignore... we all good?